pkgs/by-name/ve/vex-tui/package.nix

```
@@ -0,0 +1,25 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  lib,
+}:
+
+buildGoModule rec {
+  pname = "vex-tui";
+  version = "1.1.1";
+  src = fetchFromGitHub {
+    owner = "CodeOne45";
+    repo = "vex-tui";
+    rev = "v${version}";
+    hash = "sha256-rxpC+WyBlDELT8cfnTVkW8V6+z1mIultGhw/DoFOblQ=";
+  };
+
+  vendorHash = "sha256-PvaV0tJjIVppB36Cxg4aAKX0MBjgFC5S4GTs1zHxCCU=";
+
```

the author [intends](https://github.com/CodeOne45/vex-tui/blob/f698eb0bc0f4804c0d944a111c4736d79b30347a/Makefile#L11)[^1] that the executable is to be named `vex`, let's follow that
```suggestion
  postInstall = ''
    mv $out/bin/{vex-tui,vex}
  '';
```
sadly, `GOFLAGS = ["-o" "vex"];` can't be used, it throws an error